src/qml/views/timeline/timeline.qml

```diff
@@ -232,6 +235,103 @@ Rectangle {
                                 currentTrack = index
                                 timeline.selectTrackHead(currentTrack)
                             }
+                            MouseArea {
+                                id: dragMouseArea
+                                anchors.top: parent.top
+                                anchors.bottom: parent.bottom
+                                anchors.left: parent.left
+                                hoverEnabled: true
+                                cursorShape: Qt.DragMoveCursor
+                                drag.target: dragItem
+                                drag.axis: Drag.YAxis
+                                drag.maximumY: trackHeaders.height - y - height / 2
+                                drag.minimumY : -y - height / 2
+                                width: containsMouse ? trackHead.width : 8
+                                Behavior on width { PropertyAnimation {easing.type: Easing.InOutCubic} }
+                                onReleased: {
+                                    if (drag.active) {
+                                        dragItem.Drag.drop()
+                                    }
+                                }
+                                Rectangle {
+                                    id: dragItem
+                                    property var trackHead: trackHead
+                                    color: activePalette.dark
```

Comment:
The palette dark color is rather undefined but I see it as a mid-gray. I think it should be the same as the the block that appears when you drag a clip to the timeline, which is white with 50% opacity. However, 50% might be too translucent here. So, I think this color is close enough. But I am wondering if this should be the same solid color or some slightly alternating shade like we do for the track background color?